Sources/BraintreeVenmo/BTVenmoClient.swift

```diff
@@ -100,7 +99,7 @@ import BraintreeCore
             }
             
             do {
-                _ = try self.verifyAppSwitch(with: configuration, fallbackToWeb: request.fallbackToWeb)
+                _ = try self.verifyAppSwitch(with: configuration)
```

Comment:
Does verifyAppSwitch need to bother with returning any value?

Reply:
I assumed it would still need to return `true` since `verifyAppSwitch` checks if Venmo is enabled and if Venmo app is installed. Let me know if you think it's ok to remove the Bool return otherwise

Reply:
If ```verifyAppSwitch``` is only being used here I think it should be safe. I think the function will just throw an error  if not enabled, and the returned value isn't being used.